src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+// Get table and it's raws from HTML
+const table = document.querySelector('table');
+const rows = table.rows;
+
+// Create array from the copied data
+const columnToCopy = 1;
+const copiedData = [];
+
+for (let i = 0; i < rows.length; i++) {
+  const cell = rows[i].cells[columnToCopy];
+
+  copiedData.push(cell ? cell.textContent : '');
+}
+
+// Creating new row in HTML and insering data
+for (let i = 0; i < rows.length; i++) {
+  const newCell = rows[i].insertCell(rows[i].cells.length - 1);
+
+  newCell.textContent = copiedData[i];
+}
+
+// Receiving collection from the first table raw
+const firstRow = table.rows[0].cells;
+
+// Changing tag of the new row header from <td> to <th>
+const newRowHeader = firstRow[firstRow.length - 2];
```

Comment:
The variable `newRowHeader` is intended to select the last cell of the first row, but it uses `firstRow.length - 2`. This might not select the correct cell if the intention is to modify the newly added column header. Ensure that the index correctly targets the new header cell.